### PR TITLE
feat(contracts): NFT conviction voting power & Hypercerts Pool

### PR DESCRIPTION
**Pull request by Oba-One**
## Summary

Implements Phases 1-2 of NFT-based conviction voting power for the CV strategy Diamond.

- **NFTPowerRegistry**: Aggregates voting power from multiple token sources (ERC-721, ERC-1155, Hats Protocol) with configurable weights via `IVotingPowerRegistry`
- **PointSystem.Custom**: New enum variant enabling external registry-based power lookup instead of built-in token staking
- **CVSyncPowerFacet**: Diamond facet for syncing conviction when NFT ownership changes (`syncPower` + `batchSyncPower`) with staleness checks
- **EIP-7201 namespaced storage**: `CVSyncPowerStorage` and `CVRegistryWhitelistStorage` for collision-safe Diamond storage
- **Registry whitelist**: `CVAdminFacet` gains add/remove/check for trusted voting power registries
- **Community integration**: `CommunityMemberFacet` and `CommunityPowerFacet` updated for registry-aware member validation and power calculation

## Test plan

- [x] 18 NFTPowerRegistry unit tests (all power source types, multi-source aggregation, edge cases)
- [x] 11 CVSyncPowerFacet tests (sync, batch sync, staleness, authorization)
- [x] 6 existing test suites adapted for new `PointSystem.Custom` interface
- [x] Full suite: 610/610 pass, 0 failures

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: pkg/contracts/src/CVStrategy/facets/CVSyncPowerFacet.sol [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        }
+    }
+
+    /*|--------------------------------------------|*/
+    /*|              INTERNAL HELPERS              |*/
+    /*|--------------------------------------------|*/
+
+    function _syncPowerInternal(address _member) internal {
+        // Read cached power from community registry
+        uint256 oldPower = registryCommunity.getMemberPowerInStrategy(_member, address(this));
```
[the remark this answers is not in the document]
Comment (Oba-One) [reply pkg/contracts/src/CVStrategy/facets/CVSyncPowerFacet.sol]: Fixed. Added per-member sync baseline caching in CVSyncPowerStorage (syncedPower + hasSyncedPower) and now syncPower uses that cached baseline before applying deltas, preventing repeated old-power reapplication.
